api-reference/10 UI Components/dxStepper/1 Configuration/selectOnFocus.md

```diff
@@ -5,7 +5,7 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies if items are selected automatically when focused via keyboard.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+If this property is `true`, arrow keys move focus and select the step. If `false`, steps remain unselected when navigated with arrow keys.
```

Comment:
The current description leaves some things to interpretation... For example you say "steps remain unselected". But if they we previously selected, then they remain "selected". Or you say "move focus and select the step". But if I press Left Arrow, then steps can get deselected. 

So I want to use the phrase "synchronized selection and navigation". 

```suggestion
Users can press arrow keys to move focus between items. Set `selectOnFocus` to `true` to synchronize keyboard navigation and item selection. 
```